src/Sentry.AspNetCore/SpanStatusMapper.cs

```diff
@@ -0,0 +1,23 @@
+using Sentry.Protocol;
+
+namespace Sentry.AspNetCore
+{
+    internal static class SpanStatusMapper
+    {
+        public static SpanStatus FromStatusCode(int statusCode) => statusCode switch
+        {
+            400 => SpanStatus.InvalidArgument,
+            401 => SpanStatus.Unauthenticated,
+            403 => SpanStatus.PermissionDenied,
+            404 => SpanStatus.NotFound,
+            409 => SpanStatus.AlreadyExists,
+            429 => SpanStatus.ResourceExhausted,
+            499 => SpanStatus.Cancelled,
+            500 => SpanStatus.InternalError,
+            501 => SpanStatus.Unimplemented,
+            503 => SpanStatus.Unavailable,
+            504 => SpanStatus.DeadlineExceeded,
+            _   => SpanStatus.UnknownError
```

Comment:
Does this intentionally ignore the non-error case? 200 => Ok.

Compare with [tracing.py#L319-L350](https://github.com/getsentry/sentry-python/blob/f6f3525f8812f60911573a8b7f71807ecf2e1052/sentry_sdk/tracing.py#L319-L350) for reference.